File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/web/Models/OrderViewModel.cs
using System;
using System.Collections.Generic;

namespace Registration.Web.Models
{
    public class OrderViewModel
    {
        public OrderViewModel()
        {
            this.Items = new List<OrderItemViewModel>();
            this.Seats = new List<SeatQuantity>();
        }

        public Guid OrderId { get; set; }
        public int OrderVersion { get; set; }
        public Guid ConferenceId { get; set; }
        public string ConferenceCode { get; set; }
        public string ConferenceName { get; set; }
        public IList<OrderItemViewModel> Items { get; set; }
        public IList<SeatQuantity> Seats { get; set; }
        public long ReservationExpirationDate { get; set; }
    }
    public class SeatQuantity
    {
        public Guid SeatType { get; set; }
        public int Quantity { get; set; }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/controller/OrderViewModel.java
package com.microsoft.conference.registration.controller;

import lombok.Data;

import java.util.List;

@Data
public class OrderViewModel {
    private String orderId;
    private int orderVersion;
    private String conferenceId;
    private String conferenceCode;
    private String conferenceName;
    private List<OrderItemViewModel> items;
    private List<SeatQuantity> seats;
    private long reservationExpirationDate;
}
